bin/alluxio-start.sh

```diff
@@ -131,8 +131,17 @@ do_mount() {
   MOUNT_FAILED=0
   case "$1" in
     Mount|SudoMount)
-      ${LAUNCHER} "${BIN}/alluxio-mount.sh" $1
-      MOUNT_FAILED=$?
+      local tier_path=$(${BIN}/alluxio getConf alluxio.worker.tieredstore.level0.dirs.path)
```

Comment:
From a user perspective, if we allowed passing the Mount/SudoMount options even when the level0 tier isn't MEM I feel like it can lead to confusion on whether something is actually being mounted. I would opt for printing an error and `exit 1` in that case.